risc0/circuit/bigint/examples/rsa.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num_bigint::BigUint;
+use num_traits::Num as _;
+use risc0_circuit_bigint::zkr::register_zkrs;
+use risc0_circuit_bigint_test_methods::RSA_ELF;
+use risc0_zkvm::{ExecutorEnv, ExecutorImpl};
+
+fn main() {
+    tracing_subscriber::fmt()
+        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .init();
+
+    register_zkrs();
```

Comment:
this doesn't have to be addressed right now at all but I wonder if there's a way to avoid having the host-side code call this and instead have it included in some sort of existing sequence of functions that the host side already calls. it seems awkward for the user to have to call this in something like a starter template... one place would be to call this function when building the executor env but I'm not sure that's a great place for something like this